src/analysis/ControlFlowGraph.h

```diff
@@ -110,6 +115,16 @@ namespace vc4c
 		ControlFlowLoop findLoopsHelper(const CFGNode* node, FastMap<const CFGNode*, int>& discoveryTimes, FastMap<const CFGNode*, int>& lowestReachable, RandomModificationList<const CFGNode*>& stack, int& time);
 	};
 
+	/*
+	 * A relation in the control-flow-loop
+	 */
+	struct LoopInclusion
+	{
+		bool includes;
+		LoopInclusion(bool _includes) : includes(_includes) {}
+	};
+	using LoopInclusionTreeNode = Node<ControlFlowLoop*, LoopInclusion>;
+
```

Comment:
Is there a reason, you define the graph-node and relation types (`LoopInclusionTreeNode` and `LoopInclusion`) in the header, but the graph-type (`LoopInclusionTree`) in the cpp-file?